### DISCUSSION TITLE
A few additions to add a sense weight and impact.

### DISCUSSION THREAD

**Discussion by GrumbleGrits** [zealsprince/lc-locker#6] [Ideas]
Despite its size and the ferocity of its chase the Locker hardly ever overshoots its target since it can stop near instantly, which makes for a somewhat... unusual, weightless feel to the enemy. If the Locker retained its momentum, allowing it to overshoot its target, alongside having a tangible impact when ramming into obstacles(similar to the Crawler/Thumper), it would immediately feel much more impactful while simultaneously improving the ability to bait it into running over other creatures or even trapping itself.

**Comment by zealsprince** [comment]
I would love to do this but right now lack the time to look into how to implement that. I'd absolutely take a lot of the Thumper's logic for accomplishing this though so it's something I can look into later. Now that I think about it, it's probably easier to accomplish this than I'm figuring right now.

I can look into this say a month from now to see how it would effect gameplay as well. I want to keep that thrill of it snapping closed right in front of you as well but I do understand it adds to the humor of it when it overshoots and rams into someone else.

**Reply by GrumbleGrits** [reply]
I can see the appeal of having it stop just short of someone, snapping shut in their face, and the quick stop helps any possible follow-up charge, though with all the side paths and obstacles I find that the Locker rarely ever gets the chance to do that envisioned narrow miss face-to-face. Whenever it does, it usually resulted in me being trapped against the wall right next to it with no ability escape any follow-up.

The overshot/momentum could be based on distance traveled, so short charges remain quick and snappy while longer ones can overshoot the target.

When I imagine it overshooting its destination, I don't only imagine it hitting someone unintended. A pretty amusing scenario, imo, could arise from watching someone else getting ran over by it in a hallway intersection only for it to keep going a short ways out of sight. a hit and run, lol.
